src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -16,7 +16,6 @@
   @mixin eds-theme-typography-button-label;
   display: inline-flex;
   align-items: center;
-  justify-content: center;
```

Comment:
seems like this style came with the BM files originally, and threw off some styling in traject